src/css/components/menu/_menu-inline.scss

```diff
@@ -0,0 +1,35 @@
+.video-js .vjs-menu-button-inline {
```

Comment:
I know it's a lot of CSS, but seems better to me to keep all the menu styles in one file. Just my opinion.